sbt-plugin/src/main/scala/org/scalajs/sbtplugin/LinkerImpl.scala

```diff
@@ -77,7 +77,36 @@ object LinkerImpl {
       "scala.",
       "org.scalajs.linker.interface.",
       "org.scalajs.logging.",
-      "org.scalajs.ir."
+      "org.scalajs.ir.",
+      /*
+       * A workaround for the OpenJDK bug 6265952 (#3921)
+       * https://bugs.java.com/bugdatabase/view_bug.do?bug_id=6265952
+       *
+       * It manifests as a `java.lang.NoClassDefFoundError` being thrown,
+       * claiming the class `MethodAccessorImpl` is missing. The package of the
+       * class is implementation specific. The currently known packages are
+       * listed as prefixes below.
+       *
+       * The bug is triggered when calling `java.lang.Method#invoke` if both of
+       * the following conditions are met:
+       *
+       * - this call 15th (or later) time `invoke` is called on this instance,
```

Comment:
This sentence looks broken.

Otherwise LGTM.

Reply:
:-/ yes. fixed